sources/aft-lambda-layer/aft_common/aft_utils.py

```
@@ -3,6 +3,7 @@
 #
 import logging
 import random
+import re
```

Ruff (0.8.2)

`re` imported but unused

Remove unused import: `re`

(F401)